### PR TITLE
Text styles

### PR DESCRIPTION
**Pull request by Ben-Graham**

### CODE REVIEW

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
+
+= AsciiDoc
+
+== Headers
+
+Writing an equals sign before a line of text will make it into a header
+
+----
+= A Desciptive Title
```
Comment (bill-luu) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Typo in descriptive

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 2]
```diff
@@ -77,7 +77,55 @@ For example:
...
+
+----
+*Bolded Statement*
+----
+
+Will appear as:
+
+*Bolded Statemenet*
+
+Individual parts of a word can be bolded by breaking up with word with double asterisks.
```
Comment (bill-luu) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: `breaking up with word with double asterisks` -> `breaking up the word with double asterisks`

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+== Source Code
+
+To write source code in asciidoc, write "[source,a language]"
+Lines after the square brackets will be displayed as a block of code.
+An empty line will end the block of code.
+For example:
+
+----
+[source, c]
+//Some c code
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Surrounding with `----` is our preferred way of doing it. Please show it as such.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 4]
```diff
@@ -0,0 +1,140 @@
+
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Extra space.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+To bold text in an asciioc place a single asterisk on each side of the phrase or word to be bolded.
+
+----
+*Bolded Statement*
+----
+
+Will appear as:
+
+*Bolded Statemenet*
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Typo

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+Some Code
+----
+-----
+
+
+== Formatted Text
+
+=== Bold Text
+
+To bold text in an asciioc place a single asterisk on each side of the phrase or word to be bolded.
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Typo

**Pull request merged**